README.md

```diff
@@ -10,9 +10,11 @@ This repo includes three different custom targets you can use based on the `NUCL
 
 ## How to set up this project:
 
-1. Clone it to your machine.  Don't forget to use `--recursive` to clone the submodules: `git clone --recursive https://github.com/mbed-ce/mbed-ce-custom-targets.git`
+1. Clone it to your machine.  Don't forget to use `--recursive` to clone the submodules: `git clone --recursive https://github.com/mbed-ce/mbed-ce-custom-targets.git`and update MbedOS with `git submodule update --remote mbed-os` (call it in project folder)
```

Comment:
Nitpick: "(call it in project folder)" -> "(run this command from the mbed-ce-custom-targets folder)"